docs/specification/README.md

```diff
@@ -140,6 +140,7 @@ The first record in every mcap file is a header.
 | Bytes | Name     | Type                      | Description                                                                                                                                                                                                                                                                                                                                           |
 | ----- | -------- | ------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | 4+n   | profile  | String                    | The profile to use for interpretation of channel info user data. If the value matches one of the [supported profiles][profiles], the channel info user data section should be structured to match the description in the corresponding profile. This field may also be supplied empty, or containing a framework that is not one of those recognized. |
+| N | library | String | freeform string for writer or agent to specify its name and version |
```

Comment:
```suggestion
| N | writer | String | freeform string for writer to specify its name, version, or other information for use in debugging |
```

Reply:
Why writer?

Reply:
_writer_ to me could mean the organization? Or maybe I am overthinking it.

Reply:
It's the program that wrote the data. Another idea is "creator": https://en.wikipedia.org/wiki/Creator_code

Reply:
From John:

> i like library because even though it risks being pedantically wrong (my writer is just a single .exe file, no library involved!) it's hard to confuse it with another concept like organization name or a person's name

Reply:
Updated the description here: https://github.com/foxglove/mcap/pull/13